content/docs/command-reference/metrics/add.md

```diff
@@ -68,22 +62,24 @@ Let's first create a regular <abbr>output</abbr> with the `-o` option of
 `dvc run`:
 
 ```dvc
-$ dvc run -o metrics.txt "echo 0.9643 > metrics.txt"
+$ dvc run -o metrics.json \
+        'echo {\"AUC\": 0.9643, \"TP\": 527} > metrics.json'
```

Comment:
I'm not sure the JSON example is a better introductory example than raw, especially since there's escape characters in the string, may not be obvious what's going on to beginners.

Reply:
Maybe we should keep both separately?

Reply:
Also should we use `-O` here if we want to encourage people to track metric files with Git later?

Reply:
@jorgeorpinel we need to get rid of the raw format.
Raw format is only convenient in examples. People use json/yaml in real life. Also, the raw format confuses users - forces them to not usual behavior.

Reply:
OK sounds good. I still see `raw` listed in the command refs in this PR though, should we remove it. And what about my question about using -O (capital o) here? Please lmk if/when you need help taking over this PR.

Reply:
@jorgeorpinel good point. fixed.